### PR TITLE
more-info for thermostat-card

### LINKED ISSUE #2581: Allow opening more info in Thermostat card
Add a triple dot button in top right of the card that opens the more info dialog for the thermostat.

### PR DESCRIPTION
**Pull request by iantrich**
image

closes https://github.com/home-assistant/home-assistant-polymer/issues/2581

### CODE REVIEW

**Comment by iantrich** [comment]
If put in horizontal stacks,, things could start looking strange. Should I add an option to not render this?

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 1]
```diff
@@ -533,6 +545,13 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           vertical-align: top;
           margin-left: var(--uom-margin-left);
         }
+        .more-info {
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-thermostat-card.ts]: Color it with the text color, which is not a full black. Maybe even secondary text color. We don't want it to attract too much attention

**Comment by balloob** [comment]
Let's not start with it as an option just yet.

**Comment by bramkragten** [comment]
I think this fits the card better...
image

**Comment by iantrich** [comment]
That does look better

**Comment by balloob** [comment]
It looks better but it is confusing UX. There are 4 buttons that represent operation modes and toggle the thermostat as such. The 5th button opens a dialog.

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
Not merging it yet until we resolve the discussion 👍

**Comment by balloob** [comment]
Actually will merge it, we can keep discussing and improving.

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 2]
```diff
@@ -125,6 +126,11 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           small: !this._broadCard,
         })}">
         <div id="root">
+          <ha-icon
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-thermostat-card.ts]: This should have been a `<paper-icon-button>`.

**Pull request merged**